test/test_circuit_assert_ancilla_is_zero.py

```diff
@@ -12,9 +12,9 @@ class TestCircuitAssertAncillaIsZero(unittest.TestCase):
     $ pwd
```

Comment:
In the test files, the changes are basically
- test_circuit -> circuit
- qasm -> circuit
- qiskit_circuit -> circuit

only.